### PR TITLE
NameIt

### PR DESCRIPTION
**Pull request by Robik81**
Ability to carve a label to your stuff to personalize it.

nameit_01
nameit_02
nameit_03
nameit_04
nameit_05
nameit_06

### CODE REVIEW

**Comment by Zireael07** [comment]
Cool!

**Comment by Marrim** [comment]
Does the name appear on the layering sorting list too?

**Comment by Robik81** [comment]
@Marrim No, labeled wearables are listed by their type name (Holy Helmet as army helmet from an example above) in layering screen.

**Comment by chaosvolt** [comment]
Ooh, interesting concept.

Now if only I could write names on monsters. o3o

**Comment by pouar** [comment]
Awesome

File: src/output.cpp [hunk 1]
```diff
@@ -1194,15 +1194,19 @@ std::string format_item_info( const std::vector<iteminfo> &vItemDisplay,
...
                    std::vector<iteminfo> &vItemDisplay, std::vector<iteminfo> &vItemCompare,
                    int &selected, const bool without_getch, const bool without_border, const bool handle_scrolling)
 {
     std::ostringstream buffer;
     int line_num = 1;
-    if (sItemName != "") {
-        buffer << sItemName << "\n \n"; //This space is required, otherwise it won't make an empty line.
+	if (sItemName != "") {
+		buffer << sItemName << "\n";
+	}
```
Comment (BevapDin) [inline src/output.cpp]: There are tabs here, and in iuse_actor.cpp. Those are not allowed, please replace them with 4 spaces per tab.

**Comment by chaosvolt** [comment]
The Tab Bandits strike again. ;w;

**Comment by Robik81** [comment]
@BevapDin Sorry about that, made it in freshly installed VS 2015. I tried to fix styling when I realized, but apparently failed.

**Pull request merged**